_inc/client/engagement/index.jsx

```diff
@@ -101,6 +110,25 @@ export const Engagement = ( props ) => {
 						toggling={ isTogglingModule( element[0] ) }
 						toggleModule={ toggleModule } />;
 		}
+
+		if ( isPro && props.sitePlan.product_slug !== 'jetpack_business' ) {
+			proProps = {
+				module: element[0],
+				configure_url: ''
+			};
+			toggle = <ProStatus proFeature={ element[0] } siteAdminUrl={ props.siteAdminUrl } />;
+
+			element[1] = <span>{ element[1] }<Button compact href="#/plans">{ __( 'Pro' ) }</Button></span>;
+		}
+
+		if ( element[0] === 'google-analytics' && sitePlan.product_slug !== 'jetpack_business' ) {
```

Comment:
I think we'll need to support monthly plans as well. They use the `jetpack_business_monthly` slug.